guide/interactions/registering-slash-commands.md

```diff
@@ -93,45 +128,147 @@ await rest.put(
 );
 ```
 
+::: warning
+Remember what we mentioned earlier about command's names:
+
+- Your bot **can** have a global and guild command with **the same name**
+
+If you register the a slash command as Guild command **and** as Global command, you will see "duplicates" commands in your guild. One will be the guild command and the other will be the global command.
+:::
+
+## Editing or deleting slash commands
+
+Once your commands have been registered, they will "stay" in Discord. Anything you do in your local files after this point will **not** affect the structure of the commands that have been registered already. You can edit the structure of a slash command or delete an existing slash command by registering a "new set of commands".
+
+With this in mind, it's really straightforward how to edit or delete slash commands.
+
+### Edit
+
+To edit a slash command, simply change the structure of an existing slash command, and run the deployment script again to register the "new set of commands".
+
+Example: Let's edit the `ping` command by changing the `description` of it.
+
+Before:
+```js {6}
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('ping')
+		.setDescription('Replies with Pong!'),
+	async execute(interaction) {
+		await interaction.reply('Pong!');
+	},
+};
+```
+
+After:
+```js {6}
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('ping')
+		.setDescription('Replies with Pong!!!'),
+	async execute(interaction) {
+		await interaction.reply('Pong!');
+	},
+};
+```
+
+Finally, run the following command again to register the "new set of commands":
+```sh:no-line-numbers
+node deploy-commands.js
+```
+
+### Delete
+
+To delete a slash command, simply exclude it from the array that gets passed to `rest.put()`, and run the deploymnent script again to register the "new set of commands".
+
+#### Delete one command
+
+Let's delete the `user` command as an example. You simply exclude it from the `commands` array in the deployment script. Since we are using the command handler to load all the command files that are inside the `commands` folder, this means we just need to delete the `user.js` file from the folder, or simply move it to another directory.
```

Comment:
```suggestion
Let's delete the `user` command as an example. You simply exclude it from the `commands` array in the deployment script. Since you are using the command handler to load all the command files that are inside the `commands` folder, this means you just need to delete the `user.js` file from the folder, or simply move it to another directory.
```